src/components/date-picker/DatePicker.vue

```diff
@@ -144,9 +144,10 @@ export default class DatePicker extends Vue {
 
   /** Emit date to add or remove. */
   @Emit('emitDate')
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  protected emitDate (date: string): void {
+  protected emitDate (): string {
     this.displayPicker = false
+    // Convert to ISO string and truncate to just date portion
+    return this.date?.toISOString()?.slice(0, 10)
```

Comment:
I think you should use the DateMixin (or DateUtilities) function to convert this.

The problem is that a UTC/GMT/ISO string is 7 or 8 hours ahead of Pacific time, so you will get the wrong value here for the first 7 or 8 hours of a day (it will be ahead by 1 day).

Reply:
You're right, it would've been the wrong value early in the day. I've replaced the `toISOString` calls I used with functions from DateMixin. However, using the mixin still results in the previous problem of the displayed date being wrong since the picker defaults to 00:00:00 GMT time which, when converted to PST by the mixin, puts the displayed value back a day.

I think the previous solution of using `toLocaleDateString` without converting the timezone would work, but I'm not sure if it should go in the util or in the component itself.

Reply:
Oh, I didn't realize the picker sets it to 0Z :(

As this is specific to this component, I think you should work around it in here. My suggestion is to create a computed (local getter) in here to provide a yyyy-mm-dd that you expect (ie, yes, truncate the ISO date if that's easiest). Then you can use this computed to emit the events.

Reply:
I've fixed it following this approach of using a local getter and I think it it works pretty well, even if it's partially duplicated from the mixin.

Reply:
Looks good to me. Pretty clean!